code/game/area/Space Station 13 areas.dm

```diff
@@ -849,6 +849,48 @@ proc/process_adminbus_teleport_locs()
 	name = "Ghetto Bar"
 	icon_state = "ghettobar"
 
+//Defficiency
+
+/area/maintenance/vault
+	name = "Vault Maintenance"
+	icon_state = "vaultmaint"
+
+/area/derelictparts/fsderelict

```

Comment:
![ss 2015-02-03 at 01 00 27](https://cloud.githubusercontent.com/assets/5224390/6011786/10c31bb2-ab40-11e4-8161-45a7f973c57a.png)
What better word is there?